e2e/terraform/shared/config/provision-client.sh

```diff
@@ -46,7 +46,6 @@ sudo chown -R $HOME_DIR:$HOME_DIR /home/$HOME_DIR/examples
 sudo chmod -R 775 /home/$HOME_DIR/examples
 
 # Set env vars for tool CLIs
-echo "export VAULT_ADDR=http://$IP_ADDRESS:8200" | sudo tee --append /home/$HOME_DIR/.bashrc
```

Comment:
curious why is this line no longer needed.   Is it because it's already the default?

Reply:
There's no vault on the client nodes, so it never worked!